acis_thermal_check/main.py

```diff
@@ -1393,6 +1286,23 @@ def write_index_rst(self, outdir, context):
         # Render the template and write it to a file
         with open(outfile, "w") as fout:
             fout.write(template.render(**context))
+        if self.msid == "fptemp":
+            # For the ACIS FP limit we write out an obsid table
+            context["acis_obs"] = self.acis_and_ecs_obs
+            template_path = (
+                TASK_DATA / "acis_thermal_check/templates/obsid_template.rst"
```

Comment:
I don't see where TASK_DATA is defined.

Reply:
In passing - I think it is imported from utils here https://github.com/acisops/acis_thermal_check/blob/4f7c496bb51f327b16a75066896d5c5e5dfbe55d/acis_thermal_check/main.py#L25